e2e/pages/abstract_page.py

```diff
@@ -26,7 +26,6 @@ def load(self) -> None:
         expect.set_options(timeout=E2E_TIMEOUT)
         response = self._page.goto(self.URL)
         assert response.status == 200
-        self._page.wait_for_timeout(1000)
```

Comment:
Removed `wait_for_timeout()` and used `time.sleep()` instead and only if its necessary, since the documentation of playwright suggests to not use it and I also dont know what else `wait_for_timeout` is doing, probably not good and increasing flakyness as they say.

Reply:
In this case, I added a new locator that waits for the backend to be ready before proceeding to login.